scripts/release-image.sh

```diff
@@ -31,6 +31,7 @@ release-arm() {
 }
 
 release-ramalama() {
+    digest=$(podman image inspect "${REPO}/$1" --format '{{ .Digest }}' | cut -f2 -d':')
```

Comment:
I don't know how this script is used but here digest is not used ?

Reply:
Oops, cut and paste failure. Added missing line.